doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -33,48 +35,93 @@ Generated keys default to retaining the seed used.
 The seed is also by default retained when keys are loaded from B<PKCS#8> files
 in the seed format.
 When available, the seed parameter is also used during key export and import,
-with keys regenerated from the seed even if separately provided on import.
+with keys (by default) regenerated from the seed even when also provided on import.
+See L</Provider configuration parameters> below for related controls.
 
 When the seed is retained, it is also available as a B<gettable> parameter,
-and private key output to B<PKCS#8> files will be in seed format.
+and private key output to B<PKCS#8> files will default to seed format.
 When the seed is not available, because not included in the B<PKCS#8> file, not
 available on import, or not retained, B<PKCS#8> private key files will have the
-private key in FIPS 203 C<dk> format, without DER octet-string wrapping.
+private key in FIPS 203 C<dk> format.
+
+=back
+
+=head2 Common parameters
+
+In addition to the common parameters that all keytypes should support (see
+L<provider-keymgmt(7)/Common Information Parameters>), B<ML-KEM> keys
+keys support the following.
+
+=over 4
+
+=item "pub" (B<OSSL_PKEY_PARAM_PUB_KEY>) <octet string>
+
+The public key value.
+
+This parameter is used when importing or exporting the public key value with
+the EVP_PKEY_fromdata() and EVP_PKEY_todata() functions.
+The value format is that of the FIPS 203 (Algorithm 16:
+B<ML-KEM.KeyGen_internal>) B<ek> public key.
```

Comment:
This is repeated immediately below and seems redundant.  Please remove so this bit is consistent with text about the private key.